tests/filtertransformers/test_mongo.py

```diff
@@ -446,6 +446,19 @@ def test_other_provider_fields(self, mapper):
             "_other_provider_field": {"$gt": 1}
         }
 
+    def test_reference_structure(self, mapper):
+        """Test that fields from other providers generate
+        queries that treat the value of the field as `null`.
+
+        """
+        from optimade.filtertransformers.mongo import MongoTransformer
```

Comment:
I think this docstring needs to be updated.

Reply:
I have updated the docstring for the test.